accounts/models.py

```diff
@@ -40,6 +40,7 @@
     StudyPermission,
 )
 
+from django.utils.translation import ugettext_lazy as _
```

Comment:
why did we make the decision to move to `ugettext_lazy` here?  Everywhere else is `gettext_lazy`.  Additionally, [this stack overflow](https://stackoverflow.com/a/4164683) post suggests that the lazy variant shoudn't be used in models.  What has been your experience?